charts/amazon-cloudwatch-observability/values.yaml

```diff
@@ -242,6 +242,318 @@ containerLogs:
             log_stream_prefix   ${HOST_NAME}.
             auto_create_group   true
             extra_user_agent    container-insights
+      adcIsoExtraFiles:
```

Comment:
If the only difference is the endpoint, could we figure out a way to template it out instead of duplicating each of the configs?

Reply:
Ideally, we would fix this in fluent bit so the right endpoint would be used without having to override it https://github.com/fluent/fluent-bit/blob/master/src/aws/flb_aws_util.c#L75, but until that's done, I'm fine with doing it this way.